site.deploy.yml

```diff
@@ -24,22 +36,28 @@
       changed_when: false
       failed_when: >-
         ansible_distribution not in targets.keys()
-        and (ansible_distribution_version not in target_versions
-        or ansible_distribution_major_version not in target_versions)
+        or (ansible_distribution_release not in target_versions
+        and ansible_distribution_version not in target_versions)
       vars:
         targets:
           CentOS: ["7.6", "7.7", "8.0", "8.1"]
-          Debian: ["8", "9", "10"]
-          Ubuntu: ["16.04", "18.04"]
+          Debian: ["jessie", "stretch", "buster"]
+          Ubuntu: ["xenial", "bionic"]
         target_versions: "{{ targets[ansible_distribution] }}"
 
+    # Check if this host is managed by Ansible by the existance
+    # of the file '/etc/ansible/wordpress-ultrastack'.
+    # If the file doesn't exist, ansible_deployed.changed is True.
```

Comment:
Would you prefer to have comments on tasks such as this?  From my reading of the task `name:` some of these may not need additional clarification.

If they are helpful my current opinion is to add them, though my goal is to try to make the `name:` of a task concise enough to convey what's being performed.